docs/sugar/03-working-with-sugar.md

```diff
@@ -0,0 +1,66 @@
+# Working with Sugar
+
+
+
+https://user-images.githubusercontent.com/26067212/165967904-0a551b6e-dc7e-4d8c-9348-3093e0d93883.mp4
```

Comment:
This should have an explanation or description of what it is instead of just dropping a random link.

A video preview would be cool but at the minimum it something like: "For a basic video tutorial you can click here"

Reply:
Got it